inspirehep/modules/search/query_factory.py

```diff
@@ -47,7 +49,7 @@
 def inspire_query_factory():
     """Create a parser returning Elastic Search DSL query instance."""
 
-    def invenio_query(pattern):
+    def invenio_query(pattern, index='records-hep'):
```

Comment:
@kaplun Not sure I understand what you mean. Passing the walkers as a parameter?